pipeline_steps/github.groovy

```diff
@@ -70,12 +70,33 @@ void add_issue_url_to_pr(){
   return null
 }
 
-
-void add_comment_to_pr(String body){
-  List org_repo = env.ghprbGhRepository.split("/")
+/**
+ * If called with add_comment_to_pr("comment"), then comment will be added to
+ * the PR that triggered this job (using ghprb env vars to discover the pr).
+ *
+ * To add comments to an alternate pr, supply repo in the form "org/repo" and
+ * prid as "id" where id is the pull request number.
+ */
+void add_comment_to_pr(String body,
+                       String repo_url="use_env",
+                       String prid="use_env",
+                       Boolean allowDuplicates=false){
```

Comment:
I like how you've added three parameters, one in snake_case, one in alllowercase and one in camelCase.